packages/hub/src/models/components/ModelCard.tsx

```diff
@@ -16,6 +16,14 @@ const Fragment = graphql`
       username
       ...UserLinkFragment
     }
+    currentRevision {
+      relativeValuesExports {
+        variableName
+        definition {
+          slug
```

Comment:
Maybe this will slow it too much? Not sure. Will be good once we get to Algolia or similar.